frontend/static/js/process-editor.js

```diff
@@ -387,34 +399,48 @@ function getMetricRowTarget(innerHTML_target, actual_target_metrics, slug) {
     });
 
     // replace null with empty strings, so that "null" is not entered in the table
-
+    if (!binary) {
     innerHTML_target['min'] = `
                         <td class="col-7" ><input type="text" name="target-minimum" id = "` + slug + `" value="` + targetValues['min'] + `"`;
     innerHTML_target['max'] = `
                         <td class="col-8" ><input type="text" name="target-maximum" id = "` + slug + `" value="` + targetValues['max'] + `"`;
-    innerHTML_target['average'] = `
-                        <td class="col-9" ><input type="text" name="target-average" id = "` + slug + `" value="` + targetValues['average'] + `"`;
+    } else {
+        innerHTML_target['min'] = `
+                        <td class="col-7" ><input type="text" name="target-minimum" id = "` + slug + `" disabled`;
+        innerHTML_target['max'] = `
+                        <td class="col-8" ><input type="text" name="target-maximum" id = "` + slug + `" disabled`;
+    }
+        innerHTML_target['average'] = `
+                        <td class="col-9" ><input type="text" name="target-average" + id = "` + slug + `" value="` + targetValues['average'] + `"`;
+
     return innerHTML_target;
 }
 
-function getMetricRowTotal(actual_target_metrics) {
+function getMetricRowTotal(actual_target_metrics, binary) {
 
     let targetTotalValue = "";
-
-    if ('total' in actual_target_metrics['target']) {
-        targetTotalValue = Math.round(actual_target_metrics['target']['total'] * 100 + Number.EPSILON) / 100;
+    if (!binary) {
+        if ('total' in actual_target_metrics['target']) {
+            targetTotalValue = Math.round(actual_target_metrics['target']['total'] * 100 + Number.EPSILON) / 100;
+        }
+    } else {
+        targetTotalValue = "-";
     }
 
     return `<td class="col-10" >` + targetTotalValue + `</td>`;
 }
 
 
 function addMinMaxToInputFields(innerHTML_target, metricData) {
-
+    let binary = metricData['binary'];
     // Rest of the innerHTML_target string
-    if (metricData['min_value'] >= 0) innerHTML_target += ' min="' + metricData['min_value'] + '"';
-    if (metricData['max_value'] >= 0) innerHTML_target += ' max="' + metricData['max_value'] + '"';
-
+    if (!binary) {
+        if (metricData['min_value'] >= 0) innerHTML_target += ' min="' + metricData['min_value'] + '"';
+        if (metricData['max_value'] >= 0) innerHTML_target += ' max="' + metricData['max_value'] + '"';
+    } else {
+        innerHTML_target += ' min="' + 0 + '"';
+        innerHTML_target += ' max="' + 100 + '"';
+    }
```

Comment:
Meiner Meinung nach brauchen wir hier keine Unterscheidung, da min und max value ja 0 und 1 sind und auf Prozessseite die Eingabe ja auch zwischen 0 und 1 erfolgen soll (eben keine Umrechnung in %, sonst müssten wir ja auch die Speicherlogik noch Anfassen zur Zurückrechnung)